tests/core/mempool/test_mempool_manager.py

```diff
@@ -251,5 +256,36 @@ async def get_coin_record(_: bytes32) -> Optional[CoinRecord]:
 
     mempool_manager = await instantiate_mempool_manager(get_coin_record)
     conditions = [[ConditionOpcode.CREATE_COIN, IDENTITY_PUZZLE_HASH, 1]]
-    _, _, result = await generate_and_add_spendbundle(mempool_manager, conditions)
+    _, _, result = await generate_and_add_spendbundle(mempool_manager, TEST_COIN, conditions)
     assert result == (None, MempoolInclusionStatus.FAILED, Err.UNKNOWN_UNSPENT)
+
+
+@pytest.mark.asyncio
+async def test_same_sb_twice_with_eligible_coin() -> None:
+    async def get_coin_record(coin_id: bytes32) -> Optional[CoinRecord]:
+        test_coin_records = {
+            TEST_COIN_ID: TEST_COIN_RECORD,
+            TEST_COIN_ID2: TEST_COIN_RECORD2,
+        }
+        return test_coin_records.get(coin_id)
+
+    mempool_manager = await instantiate_mempool_manager(get_coin_record)
+
+    sb1_conditions = [
+        [ConditionOpcode.CREATE_COIN, IDENTITY_PUZZLE_HASH, 1],
+        [ConditionOpcode.CREATE_COIN, IDENTITY_PUZZLE_HASH, 2],
+    ]
+    sb1 = spend_bundle_from_conditions(TEST_COIN, sb1_conditions)
+    sb2_conditions = [
+        [ConditionOpcode.CREATE_COIN, IDENTITY_PUZZLE_HASH, 3],
+        [ConditionOpcode.AGG_SIG_UNSAFE, G1Element(), IDENTITY_PUZZLE_HASH],
+    ]
+    sb2 = spend_bundle_from_conditions(TEST_COIN2, sb2_conditions)
+    sb = SpendBundle.aggregate([sb1, sb2])
+    sb_name = sb.name()
+    result = await add_spendbundle(mempool_manager, sb, sb_name)
+    assert result == (10268283, MempoolInclusionStatus.SUCCESS, None)
```

Comment:
what are these numbers? `10268283` it might make sense to add a comment

Reply:
Done (it's the expected cost).